src/components/BrowseResultsList/BrowseResultsList.js

```diff
@@ -65,20 +69,20 @@ const BrowseResultsList = ({
   const listId = `browse-results-list-${browseOption}`;
 
   const isSelected = useCallback(({ item, rowIndex }) => {
-    if (isRowPreventsClick(item, browseOption)) return false;
+    if (isRowPreventsClick(item, browseOption, stripes)) return false;
```

Comment:
These utility functions don't really need the whole stripes object, we can just pass a boolean value like `isNewCallNumberBrowseAvailable`
Same goes for `getBrowseResultsFormatter`